aten/src/ATen/core/ATen_pch.h

```diff
@@ -0,0 +1,148 @@
+// This global header must not depend on native_functions.yaml or
```

Comment:
is there a particular sort order to this file? was this file generated?

can you add some comments about how this was created?

Reply:
I've published my scripts in https://github.com/peterbell10/build_analysis/ and added a comment for how to recreate it.